packages/open-next/src/adapters/warmer-function.ts

```diff
@@ -27,49 +21,74 @@ const resolveWarmerInvoke = async () => {
     return await openNextParams.invokeFunction();
   } else {
     return Promise.resolve(async (warmerId: string) => {
-      const ret = await Promise.all(
-        Array.from({ length: CONCURRENCY }, (_v, i) => i).map((i) => {
-          try {
-            return lambda.send(
-              new InvokeCommand({
-                FunctionName: FUNCTION_NAME,
-                InvocationType: "RequestResponse",
-                Payload: Buffer.from(
-                  JSON.stringify({
-                    type: "warmer",
-                    warmerId,
-                    index: i,
-                    concurrency: CONCURRENCY,
-                    delay: 75,
-                  } satisfies WarmerEvent),
-                ),
-              }),
-            );
-          } catch (e) {
-            error(`failed to warm up #${i}`, e);
-            // ignore error
-          }
-        }),
+      const { InvokeCommand, LambdaClient } = await import(
+        "@aws-sdk/client-lambda"
```

Comment:
👍 nice, we should dynamically import the aws-sdk where possible.

Reply:
Yeah, the only one missing is in the image-optimization. The rest are already dynamically loaded